apollo-ios/Sources/Apollo/GraphQLExecutionSource.swift

```diff
@@ -17,6 +17,8 @@ public protocol GraphQLExecutionSource {
   /// GraphQL execution.
   associatedtype FieldCollector: FieldSelectionCollector<RawObjectData>
 
+  var shouldAttemptDeferredFragmentExecution: Bool { get }
```

Comment:
Can we add some documentation to this to explain what the implications this has on execution are.